README.md

```diff
@@ -107,6 +104,18 @@ token.transfer(
 distribute_from_whales_realtime(accounts[1], percentage=0.8)
 ```
 
+### Using distribute from whales realtime with exact amount
+```python
+import token  # some deployed token
+from brownie import accounts
+from badger_utils.token_utils.distribute_from_whales_realtime import distribute_from_whales_realtime_amount
```

Comment:
is it an idea to have one single user facing func with a simple name, path and signature? my suggestion would be `badger_utils.token_utils.whalemint(recipient, mantissa, token, perc_bool)` or something similar?

Reply:
I think it's a better API design to have 2 different functions for this purpose, but that's debatable

Reply:
Ok, refactored it as you suggested

Reply:
i guess you could always keep the 2 different functions and write a wrapper on top of it, right?

im just looking at it from a user's perspective; having one quick func makes a big difference.

Reply:
This is done